tests/unit/class-zoninator-api-controller-test.php

```diff
@@ -215,58 +215,111 @@ function test_create_zone_fail_if_invalid_data() {
-	 * T test_add_post_to_zone_responds_with_created_when_method_put
+	 * T test_delete_zone_responds_with_success_when_method_delete
 	 *
 	 * @throws Exception E.
 	 */
-	function test_add_post_to_zone_responds_with_success_when_method_put() {
+	function test_delete_zone_responds_with_success_when_method_delete() {
+		$this->login_as_admin();
+		$zone_id = $this->create_a_zone( 'test-update-zone', 'Test Zone' );
+		$response = $this->delete( '/zoninator/v1/zones/' . $zone_id );
+		$this->assertResponseStatus( $response, 200 );
+	}
+
+	/**
+	 * T test_delete_zone_responds_with_not_found_if_zone_not_exist
+	 *
+	 * @throws Exception E.
+	 */
+	function test_delete_zone_responds_with_not_found_if_zone_not_exist() {
+		$this->login_as_admin();
+		$response = $this->delete( '/zoninator/v1/zones/666666' );
+		$this->assertResponseStatus( $response, 404 );				
+	}
+
+	/**
+	 * T test_update_zone_posts_responds_with_ok_when_method_put
+	 *
+	 * @throws Exception E.
+	 */
+	function test_update_zone_posts_responds_with_success_when_method_put() {
 		$this->login_as_admin();
 		$post_id = $this->_insert_a_post();
-		$zone_id = $this->create_a_zone( 'the-zone-add-post-1', 'The Zone Add Post one' );
+		$zone_id = $this->create_a_zone( 'test-zone', 'Test Zone' );
 		$response = $this->put( '/zoninator/v1/zones/' . $zone_id . '/posts', array(
-			'post_id' => $post_id,
+			'post_ids' => array( $post_id ),
 		) );
 		$this->assertResponseStatus( $response, 200 );
 	}
 
 	/**
-	 * T test_add_post_to_zone_respond_not_found_if_zone_not_exists
+	 * T test_update_zone_posts_responds_with_not_found_if_zone_not_exist
 	 *
 	 * @throws Exception E.
 	 */
-	function test_add_post_to_zone_respond_not_found_if_zone_not_exists() {
+	function test_update_zone_posts_responds_with_not_found_if_zone_not_exist() {
+		$this->login_as_admin();
 		$post_id = $this->_insert_a_post();
 		$response = $this->put( '/zoninator/v1/zones/666666/posts', array(
-			'post_id' => $post_id,
+			'post_ids' => array( $post_id ),
 		) );
 		$this->assertResponseStatus( $response, 404 );
 	}
 
 	/**
-	 * Test test_add_post_to_zone_fail_if_invalid_post
+	 * T test_update_zone_posts_fails_if_invalid_data_format
+	 *
+	 * @throws Exception E.
 	 */
-	function test_add_post_to_zone_fail_if_invalid_post() {
-		$zone_id = $this->add_a_zone( 'zone-test_add_post_to_zone_fail_if_invalid_post' );
-		$response = $this->post( '/zoninator/v1/zones/' . $zone_id . '/posts', array(
-			'post_id' => 666666,
+	function test_update_zone_posts_fails_if_invalid_data() {
+		$this->login_as_admin();
+		$post_id = $this->_insert_a_post();
+		$zone_id = $this->create_a_zone( 'test-zone', 'Test Zone' );
+		$response = $this->put( '/zoninator/v1/zones/' . $zone_id . '/posts', array() );
+		$this->assertResponseStatus( $response, 400 );
+	}
+
+	/**
+	 * T test_update_zone_posts_fails_if_invalid_post_id
+	 *
+	 * @throws Exception E.
+	 */
+	function test_update_zone_posts_fails_if_invalid_data() {
+		$this->login_as_admin();
```

Comment:
PHPUnit Fails with the following: `PHP Fatal error:  Cannot redeclare Zoninator_Api_Controller_Test::test_update_zone_posts_fails_if_invalid_data() in <mydir>/zoninator/tests/unit/class-zoninator-api-controller-test.php on line 317`